examples/requirements.txt

```diff
@@ -4,3 +4,6 @@ scipy
 networkx
 jupyter
 matplotlib
+openfermion
+openfermionpsi4
+grove
```

Comment:
should this just be `forestopenfermion`?

Reply:
Good catch!  It seems `openfermionpsi4` is a separate project and still required, but I was able to replace `openfermion ` + `grove` with `forestopenfermion` which is also a required dependency.

Not that the user will still need to have installed the `psi4` binary out-of-band in order for `1.3_vqe_demo.py` to work. There is a conda-only binary package, but seems silly to put that in `requirements.txt`.